docs/AboutUs.adoc

```diff
@@ -4,44 +4,44 @@
 :imagesDir: images
 :stylesDir: stylesheets
 
-guiltTrip() is developed by the https://github.com/AY1920S1-CS2103-T16-4[AY1920S1-CS2103-T16-4] team. +
+guiltTrip() is developed by the https://github.com/AY1920S1-CS2103-T16-4[AY1920S1-CS2103-T16-4 team]. +
 We are a team based in the http://www.comp.nus.edu.sg[School of Computing, National University of Singapore].
 
 == Project Team
 
-=== Ong Ming Chun
+=== Ong Ming Chung
 image::dandford.png[width="150", align="left"]
 {empty}[https://github.com/Dandford[github]] [<<johndoe#, portfolio>>]
 
 Role: Team Leader +
-Responsibilities: Deliverables & deadlines
+Responsibilities: Deliverables & deadlines, Data Representation
```

Comment:
Great that the responsibilities now covered features that the member is accountable for.